packager/media/formats/mp4/mp4_media_parser_unittest.cc

```diff
@@ -103,9 +103,13 @@ class MP4MediaParserTest : public testing::Test {
 
   bool ParseMP4File(const std::string& filename, int append_bytes) {
     InitializeParser(NULL);
+
     if (!parser_->LoadMoov(GetTestDataFilePath(filename).AsUTF8Unsafe()))
       return false;
+
     std::vector<uint8_t> buffer = ReadTestDataFile(filename);
+    ASSERT_FALSE(buffer.empty());
```

Comment:
some of the introduced usages are not in void functions, such as this one. I think instead of ASSERT_FALSE we use EXPECT_FALSE, but at that point couldn't we put this inside of `ReadTestDataFile` directly?

Reply:
See PR #1247